codalab/lib/bundle_cli.py

```diff
@@ -1706,6 +1706,27 @@ def do_make_command(self, args):
             params={'worksheet': worksheet_uuid},
         )
 
+        # Add new bundle's location. If user specify the storage using `--store`, the bundle will be added to that storage.
+        # Otherwise, the new MakeBundle will be added to default storage, which is set by the rest server.
+        destination_bundle_store = metadata.get('store')
+        if destination_bundle_store is not None:
+            need_bypass = False
+            bundle_store_uuid = None
+
+            # 1) Read destination store from --store if user has specified it
+            if destination_bundle_store is not None and destination_bundle_store != '':
+                storage_info = client.fetch_one(
+                    'bundle_stores',
+                    params={
+                        'name': destination_bundle_store,
+                        'include': ['uuid', 'storage_type', 'url'],
+                    },
+                )
+                bundle_store_uuid = storage_info['uuid']
+
+            params = {'need_bypass': need_bypass}
```

Comment:
Can you also add a comment for why we should not bypass server for make bundles?

```suggestion
            params = {'need_bypass': False}
```